src/test/java/com/google/sps/UtilityTest.java

```diff
@@ -115,4 +120,114 @@ public void nullValue() {
 
     Assert.assertEquals(actual, expected);
   }
+  
+  /** 
+   * Tests for addNewUser() function.
+   * A new user is being inserted in each test, so a query to check its existence is the way to
+   * assert the functionality.
+   */
+  @Test
+  public void normalUser() {
```

Comment:
Same comment here as previous test PR: 

A common style practice for naming java unit test functions it to include the method being tested, the test condition, and the expected outcome, so something like: addNewUser_invalidMajor_doesNotExist(). This becomes especially useful for readability when you have a lot of different functions being tested within the same test file (ie. this UtilityTest file).

Doing this makes it really explicitly clear what is being tested, how it's being tested, and what we expect to see. Consider updating your methods to also follow this pattern!

It's not technically part of the public github Java style guide, so I'll leave it up to you - but a good habit to have :)